src/LevelNav.css

```diff
@@ -0,0 +1,42 @@
+.everything {
```

Comment:
I think we should use a more descriptive class name here -- maybe LevelNavContainer?